frontend/syntaxmeets/src/components/Home/Home.js

```diff
@@ -9,12 +9,32 @@ import { Link } from "react-router-dom";
 import SkyLight from "react-skylight";
 import PropTypes from "prop-types";
 import { withStyles } from "@material-ui/core/styles";
+
+import MeetingRoomIcon from '@material-ui/icons/MeetingRoom';
+import GroupAddIcon from '@material-ui/icons/GroupAdd';
+
+function generateRoomId() {
+  var tempId = "";
+  var characters = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+  var charactersLength = characters.length;
+  for (var i = 0; i < 12; i++) {
+    tempId += characters.charAt(Math.floor(Math.random() * charactersLength));
+    if ((i + 1) % 4 === 0 && i !== 11) {
+      tempId += "-";
+    }
+  }
+
+  return tempId;
+}
+
```

Comment:
this function is already present in the utils folder, please check

Reply:
It has been recently added to utils on redux commit. I'll make that change.

Reply:
@simran2607, yes and also please consider states, because after redux all states are shifted to store. So use props,state, wherever needed